integration-tests/src/test/java/oracle/kubernetes/operator/ItMiiConfigUpdateCm.java

```diff
@@ -0,0 +1,366 @@
+      assertNotNull(operator);
+      domainNS = ((ArrayList<String>) operatorMap.get("domainNamespaces")).get(0);
+      namespaceList.append((String)operatorMap.get("namespace"));
+      namespaceList.append(" ").append(domainNS);
+    }
+  }
+
+  /**
+   * Releases k8s cluster lease, archives result, pv directories.
+   *
+   * @throws Exception when errors while running statedump.sh or cleanup.sh
+   *         scripts or while renewing the lease for shared cluster run
+   */
+  @AfterAll
+  public static void staticUnPrepare() throws Exception {
+    tearDown(new Object() {
+    }.getClass().getEnclosingClass().getSimpleName(), namespaceList.toString());
+
+    LoggerHelper.getLocal().info("SUCCESS");
+  }
+
+  /**
+   * Create a domain without a JDBC DS using model in image and having config map
+   * in the domain.yaml before deploying the domain. After the domain resources
+   * is created, re-create the config map with model files that define a JDBC DataSource
+   * and patch to change domain-level restart version to reload the model files,
+   * generate new config and initiate a rolling restart.
+   */
+  @Test
+  public void testMiiConfigUpdateNonJdbcCm() {
+    Assumptions.assumeTrue(QUICKTEST);
+    String testMethodName = new Object() {
+    }.getClass().getEnclosingMethod().getName();
+    boolean testCompletedSuccessfully = false;
+    Domain domain = null;
+
+    try {
+      logTestBegin(testMethodName);
+      LoggerHelper.getLocal().log(Level.INFO,
+          "Creating Domain & waiting for the script to complete execution");
+
+      // create domain w/o JDBC DS using the image created by MII
+      boolean createDS = false;
+      domain = createDomainUsingMii(createDS, domainNS, testClassName);
+      assertNotNull(domain, "Failed to create a domain");
+
+      // copy model files that contains JDBC DS to a dir, re-create cm to update config,
+      // patch domain to change domain-level restart version and verify domain restarted successfully
+      final String destDir = getResultDir() + "/samples/model-in-image-update";
+      final String[] modelFiles = {"model.jdbc.yaml", "model.jdbc.properties"};
+      createCmAndPatchDomain(domain, destDir, modelFiles);
+
+      // get JDBC DS prop values via WLST on server pod
+      String jdbcResourceData = getJdbcResources(destDir, domain);
+
+      // verify that JDBC DS is created by checking DS name, JNDI name and read timeout
+      LoggerHelper.getLocal().log(Level.INFO, "Verify that JDBC DS is created");
+      final String[] jdbcResourcesToVerify =
+          {"datasource.name.1=" + dsName,
+              "datasource.jndiname.1=array(java.lang.String,['" + jndiName + "'])",
+              "datasource.readTimeout.1=" + readTimeout_2};
+      for (String prop : jdbcResourcesToVerify) {
+        assertTrue(jdbcResourceData.contains(prop), prop + " is not found");
+        LoggerHelper.getLocal().log(Level.INFO, prop + " exists");
+      }
```

Comment:
It looks that this JDBC DS verification logic can be an util method.

Reply:
The verification was in an util method, But mark likes to it to be clearly assert this way. Let's wait for Mark's comments here before I change it back to use an util method